### ISSUE TITLE
[Accessibility] Posts block - Posts should be in a list

### ISSUE THREAD

**Issue by andeng1106** [gambitph/Stackable#3253]
**Describe the bug**
- To be able to check this, try the WordPress Query Loop block as a sample block.
- When you add a Query Loop block, it uses the list tags:
   <img width="1660" alt="Screenshot 2024-07-19 at 10 04 46 AM" src="https://github.com/user-attachments/assets/1b253cf4-38f2-4ccc-b772-5349286c68a7">

- While Stackable Posts block uses div tags:
   <img width="1677" alt="Screenshot 2024-07-19 at 10 07 42 AM" src="https://github.com/user-attachments/assets/3c0f689b-58d6-4844-b44a-180b0c4889ff">

**To Reproduce**
1. Add a new page
2. Add a Posts block
3. Save page and view frontend
4. Inspect to see that Stackable uses div tags for each post list item

**Expected behavior**
Stackable Posts block should use list tags

**Screenshots**

https://github.com/user-attachments/assets/0d9a68b9-ef2a-4f15-a8ce-05bd7132b73e
